6.3.22/script.js

```diff
@@ -0,0 +1,33 @@
+/*------------------------------------------------------*/
+const dishes = ['Spaghetti', 'Lasagne', 'Beans on toast', 'Porridge', 'Tomato soup',
+'Salmon'];
+dishes.food = "I am hungry";
+for (let i in dishes) {
+	console.log(i);
+}
+
+for (let i of dishes) {
+	console.log(i);
+}
+/*-----------------------------------------------------*/
+const btn = document.querySelector('#btn');
+btn.onclick = () => console.log('Javascript test');
+
+function clickPlease() {
+var seltheme = document.getElementById("selcolor").value;
+   var elem = document.getElementById("pict");
+   if(seltheme == "Maroon") {
+   elem.style.backgroundColor = 'red';
+   }
+   else  if(seltheme == "Green") {
+   elem.style.backgroundColor = 'green';
+   }
+};
```

Comment:
Avoide using half words to create variable names, btn, selcolor, seltheme, pict...and also when a varianle name is compused of 2 words, in js convesion is to use camelCase, e.g. selectedColor